src/Uno.UI.RemoteControl/RemoteControlClient.cs

```diff
@@ -38,6 +38,7 @@ public class RemoteControlClient : IRemoteControlClient
 		private readonly (string endpoint, int port)[]? _serverAddresses;
 		private WebSocket? _webSocket;
 		private Dictionary<string, IRemoteControlProcessor> _processors = new Dictionary<string, IRemoteControlProcessor>();
+		private List<IRemoteControlPreProcessor> _preprocessors = new List<IRemoteControlPreProcessor>();
```

Comment:
Should these be scoped the same way _processors are?

Reply:
The main use case for this is as a way of interacting/intercepting the processing that another processor might do but without having to spread the logic across the two processors (and their assemblies.)